src/Query/Match.php

```diff
@@ -2,180 +2,17 @@
 
 namespace Elastica\Query;
 
+\trigger_error('Elastica\Query\Match is deprecated. Use Match_ instead. From PHP 8 match is reserved word and this class will be removed in further Elastica releases', \E_USER_DEPRECATED);
+
 /**
  * Match query.
  *
- * @author F21
- * @author WONG Wing Lun <luiges90@gmail.com>
+ * This class is for backward compatibility reason. For PHP 8 and above use Match_ as Match is reserved.
```

Comment:
Nice, so we can keep the old file around for compatiblity but mark it deprecated. This is great.